modules/solid_mechanics/tests/generalized_plane_strain/tests

```diff
@@ -1,38 +0,0 @@
-[Tests]
```

Comment:
@bwspenc those tests are specifically for generalized plane strain model. Since we reimplemented the model and updated the tests in tensor mechanics, should we still keep them in solid mechanics?